spec/fixtures/certs/generate.sh

```diff
@@ -12,11 +12,13 @@ echo "DO NOT USE THESE CERTIFICATES IN PRODUCTION" >> ./README.txt
 # certificate authority
 openssl genrsa -out root.key 4096
 openssl req -new -x509 -days 1826 -extensions ca -key root.key -out root.crt -subj "/C=LS/ST=NA/L=Http Input/O=Logstash/CN=root" -config ../openssl.cnf
+keytool -import -file root.crt -alias rootCA -keystore truststore.jks -noprompt -storepass 12345678
```

Comment:
As I was testing I noticed that creating the root CA p12 trust store using openssl like this:

`openssl pkcs12 -export -in root.crt -nokeys -out root_cert.p12 -name root -passout pass:root_cert`

produced a store that couldn't be read correctly, while using the same parameters with `keytool` like this:

`keytool -import -alias root -noprompt -file root.crt -keystore root_cert.p12 -storetype PKCS12 -storepass root_cert`

worked fine. was this also why you used keytool here?

Reply:
Seems to be related to https://github.com/openssl/openssl/pull/19025
and the fact that openssl wont, by default, set the trusted cert bag attribute, while the keytool does:

```
❯ openssl pkcs12 -in root_cert_openssl.p12 -nokeys -passin pass:root_cert | grep Bag
Bag Attributes: <No Attributes>
```

```
❯ openssl pkcs12 -in root_cert_keytool.p12 -nokeys -passin pass:root_cert | grep Bag -A 2
Bag Attributes
    friendlyName: root
    2.16.840.1.113894.746875.1.1: <Unsupported tag 6>
```

This goes deep into p12 vs jks and java interoperatiblity, I think it's ok for now that we stick to using the keytool to create the trust store.

Reply:
Yes, I had the same issue. I'm not pretty sure why that's happening - and TBH, didn't dig into it - but it seems that when it's generated using `openssl`, the trusted certificated is not included/readable on/from the keystore. It might be something related to how manticore reads it, encryption compatibility, etc. as it also uses the `keytool` to generate their [truststore](https://github.com/cheald/manticore/blob/master/Rakefile#L54) test files.